packages/cozy-stack-client/src/SharingCollection.js

```diff
@@ -19,8 +19,48 @@ class SharingCollection extends DocumentCollection {
       data: resp.data.map(normalizeSharing)
     }
   }
+  /**
+   *
+   * Creates a new Sharing. See https://docs.cozy.io/en/cozy-stack/sharing/#post-sharings
+   *
+   * @param {object} params
+   * @param {object} params.document The document to share
+   * @param {string} params.description Description of the sharin
+   * @param {string} params.previewPath The path (route) to see the preview
```

Comment:
I don't think that adding route helps to the comprehension, an example would be more helpful IMHO.